spacy_llm/backends/rest/backend/noop.py

```diff
@@ -3,6 +3,8 @@
 
 from .base import Backend
 
+NOOP_RESPONSE = ""
```

Comment:
Let's make this a static class variable, if there's no particular reason to have it global.